tests/integration/components/make-decision-dropdown/component-collection-test.ts

```diff
@@ -0,0 +1,169 @@
+            actions: [CollectionSubmissionActionTrigger.Accept, CollectionSubmissionActionTrigger.Reject],
+            commentLabel: 'Additional comment',
+            commentPlaceholder: 'Add remarks to project admins',
+            checkboxLabel: ['Accept Request', 'Reject Request'],
+            // eslint-disable-next-line max-len
+            checkboxDescription: ['Submission will appear in search results and be associated with the collection' , 'Submission will not appear in search results nor be associated with the collection'],
+        },
+        [CollectionSubmissionReviewStates.Accepted]: {
+            actions: [CollectionSubmissionActionTrigger.Remove],
+            commentLabel: 'Justification for Removal',
+            commentPlaceholder: 'Provide justification for removal',
+            checkboxLabel: ['Remove item'],
+            // eslint-disable-next-line max-len
+            checkboxDescription: ['Item will be removed from the collection and no longer appear in search results'],
+        },
+        [CollectionSubmissionReviewStates.InProgress]: {
+            actions: [],
+        },
+        [CollectionSubmissionReviewStates.Rejected]: {
+            actions: [],
+        },
+        [CollectionSubmissionReviewStates.Removed]: {
+            actions: [],
+        },
+    };
+
+    hooks.beforeEach(async function(this: ThisTestContext) {
+        // Given the scenario needs to be created for a collection
+        // to have a submission request in a certain review state
+
+        // When the fixture loaded
+        server.loadFixtures('licenses');
+        // And a user is created
+        const currentUser = server.create('user', 'loggedIn');
+        // And licenses are addee
+        const licensesAcceptable = server.schema.licenses.all().models;
+        // And a collection provider is created
+        const provider = server.create('collection-provider', 'currentUserIsModerator');
+        // And a collection is added with the provider
+        const mirageCollection = server.create('collection', {
+            provider,
+        });
+        // And a node with a guid is created
+        const guid = server.create('node', {
+            license: licensesAcceptable[0],
+            currentUserPermissions: [Permission.Admin],
+        }, 'withContributors');
+
+        // And a contributor is added
+        server.create('contributor', {
+            node: guid,
+            users: currentUser,
+            index: 0,
+        });
+        // And a moderator is assignede
+        server.create('moderator', { provider });
+        // And a collection submission is created
+        const mirageSubmission = server.create('collection-submission', {
+            id: guid.id,
+            creator: currentUser,
+            collection: mirageCollection,
+            guid,
+        });
+
+        // And the collectionSubmission is stored in state
+        const submissionId = mirageSubmission.id + '-' + mirageSubmission.id;
```

Comment:
Sorry I'm not super up-to-date on Collections, but wouldn't one of these be the collection id? That it's working indicates that it probably isn't, but I thought I'd check. This seems an odd `submissionId`.

Reply:
Good eye! The collection-submission's ID should be in the format `${resourceId}-${collectionId}`